omnigan/trainer.py

```diff
@@ -1321,18 +1338,20 @@ def get_masker_loss(self, multi_domain_batch):  # TODO update docstrings
                 target = batch["data"][task]
 
                 if task == "d":
-                    loss, d_pred, z_depth = self.masker_d_loss(
+                    loss_d, d_pred, z_depth = self.masker_d_loss(
                         x, z, target, domain, "G"
                     )
-                    m_loss += loss
-                    self.logger.losses.gen.task["d"][domain] = loss.item()
+                    m_loss += loss_d
+                    self.logger.losses.gen.task["d"][domain] = loss_d.item()
+                    losses.append(loss_d)
```

Comment:
@tianyu-z I don't see where the initial list of `losses` is initialized?

Reply:
Fixed